scripts/install-helpers/install.sh

```diff
@@ -175,7 +175,8 @@ function apply_operator_manifests() {
 # Function to apply the operator manifests
 function override_trustee_image() {
     if [ -n "$TRUSTEE_IMAGE" ]; then
-        oc patch csv -n trustee-operator-system trustee-operator.v0.1.0 --type=json -p="[
+        CSV=$(oc get csv -n trustee-operator-system -o name)
```

Comment:
There could be multiple CSVs present in the namespace belonging to operators which installs in all namespaces, so the above will fail. For example in my cluster
```
oc get csv -n trustee-operator-system -o name                                                    base ⎈ system:admin
clusterserviceversion.operators.coreos.com/gatekeeper-operator-product.v3.17.0
clusterserviceversion.operators.coreos.com/openshift-pipelines-operator-rh.v1.17.0
clusterserviceversion.operators.coreos.com/trustee-operator.v0.2.0
```
I think the CSV needs to be hardcoded.

Reply:
Nicely spotted. Does this work for you?
`oc get csv -n trustee-operator-system -o name -l operators.coreos.com/trustee-operator.trustee-operator-system`

Reply:
Yes.
```
oc get csv -n trustee-operator-system -o name -l operators.coreos.com/trustee-operator.trustee-operator-system
clusterserviceversion.operators.coreos.com/trustee-operator.v0.2.0
```

Btw, this approach will always install the latest version of the operator instead of pinning a specific version. Is it acceptable ?
Imho, latest should be ok. But still wanted to raise this to get your thoughts.